gnomad_qc/v4/create_release/make_var_annot_hists.py

```diff
@@ -0,0 +1,216 @@
+# noqa: D100
+
+import argparse
+import json
+import logging
+
+import hail as hl
+from gnomad.resources.resource_utils import DataException
+from gnomad.utils.annotations import create_frequency_bins_expr, get_annotations_hists
+from gnomad.utils.file_utils import file_exists
+
+from gnomad_qc.v4.resources.release import (
+    annotation_hists_params_path,
+    qual_hists_json_path,
+    release_ht_path,
+)
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+LOG10_ANNOTATIONS = ["AS_VarDP", "QUALapprox", "AS_QUALapprox"]
```

Comment:
Another question here: you got 3 annotations as log10_annotations, but why from L135-L165, you only did log 10 for the last two of them?

Reply:
Were you planning to get a histogram with their original values and with log10 scale?

Reply:
All three get passed as log in L107. The last two also get split up by frequency bin.

Reply:
Sorry I missed that. Could you add a small note before L135?